Library/Homebrew/test/.simplecov

```diff
@@ -20,7 +20,13 @@ SimpleCov.start do
     at_exit do
       exit_code = $!.nil? ? 0 : $!.status
       $stdout.reopen("/dev/null")
-      SimpleCov.result # Just save result, but don't write formatted output.
+
+      # Just save result, but don't write formatted output.
+      coverage_result = Coverage.result
+      SimpleCov.add_not_loaded_files(coverage_result)
+      simplecov_result = SimpleCov::Result.new(coverage_result)
```

Comment:
Is there a speed difference here between `SimpleCov::Result.new(Coverage.result)` and [`SimpleCov::Result.new(SimpleCov.add_not_loaded_files(Coverage.result))`](https://github.com/colszowka/simplecov/blob/342a67c986949d9b4280b0e267cd9066fe9fcdaa/lib/simplecov.rb#L83).